DeanerySystem/DeanerySystem.UI/Infrastructure/NinjectDependencyResolver.cs

```diff
@@ -1421,7 +1455,7 @@ private void AddBindings()
 
             #endregion
 
-            kernel.Bind<IDeaneryEntitiesRepository>().ToConstant(entitiesMock.Object);
+            //kernel.Bind<IDeaneryEntitiesRepository>().ToConstant(entitiesMock.Object);
```

Comment:
Do we still need all the dummy data in this file? Or can it all go away as we have fancy data providers now?

Reply:
Only fancy